src/Voron/Data/VoronStream.cs

```diff
@@ -178,6 +181,19 @@ public override int Read(byte[] buffer, int offset, int count)
             return count;
         }
 
+        private Page GetPage(long pageNumber)
```

Comment:
I have some doubts if the method name expresses what it really does

Reply:
changed